Server/package.json

```diff
@@ -10,7 +10,7 @@
   "scripts": {
     "compile": "npm run clean && tsc -p tsconfig.json",
     "clean": "rm -rf dist/* tsconfig.tsbuildinfo",
-    "start": "export NODE_DEBUG=http && export APP_ENV=local && nodemon src/index.ts",
+    "start": "export APP_ENV=local && nodemon src/index.ts",
```

Comment:
Just out of curiosity, what is this change for?

Reply:
I added this - this NODE_DEBUG=http will make sure that logs for the `typed-rest-client` are present. I believe i was using this to test a response from the client EMSP to see what the response body is. @langiole we should take note of this and probably just need to make sure to add some logs into the request handlers - I just wanted something quickly